### ISSUE TITLE
Germany ranking is broken

### ISSUE THREAD

**Issue by mustafaozhan** [ashkulz/committers.top#31]
Currently, my badge doesn't work, it sometimes happens if I do not have enough follower to be in the list.

But this time when I wanted to check if it is the case at https://committers.top/germany

I realized that something is wrong in the webpage too. I took a screenshot for clarification. It says
```
This list contains all public commits for each user. There are 0 total users in the region and you need at least -1 followers to be on this list.
```

<img width="552" alt="Screenshot 2024-06-29 at 12 50 23" src="https://github.com/ashkulz/committers.top/assets/30001999/3d7bfbaf-0311-4f5e-ad47-991994a3429f">

**Comment by ashkulz** [comment]
You're right, the automated commit a8f29a3f1d876364218e1b9bee914b38e27192db deleted all data. I'm not sure how or why that happened, need to handle it properly. Seems to have happened for b65e260897f215c0d30ad4792cf0cf67c9881f35 as well, I suspect they're part of the same run.

**Comment by ashkulz** [comment]
@mustafaozhan please check now, it should be fixed via 087c8df2cb8963bdf0f164959c8c0011d278676b for gabon and b62487d06c1e442f9fcfa7f2d4df7f40b65b80f3 for germany.

**Comment by mustafaozhan** [comment]
Confirming it is fixed, thank you @ashkulz
